File: PersistenceServer/Sep3JavaServer-master/src/main/java/PersistenceServer/Invitation.java
package PersistenceServer;

public class Invitation
{
  private int id;
  private int invitorId;
  private int inviteeId;
  private int groupId;

  public int getId()
  {
    return id;
  }
  public void setId(int id)
  {
    this.id = id;
  }

  public int getInvitorId()
  {
    return invitorId;
  }

  public void setInvitorId(int invitorId)
  {
    this.invitorId = invitorId;
  }

  public int getInviteeId()
  {
    return inviteeId;
  }

  public void setInviteeId(int inviteeId)
  {
    this.inviteeId = inviteeId;
  }

  public int getGroupId()
  {
    return groupId;
  }

  public void setGroupId(int groupId)
  {
    this.groupId = groupId;
  }

  public Invitation(int id, int invitorId, int inviteeId, int groupId)
  {
    this.id = id;
    this.groupId = groupId;
    this.inviteeId = inviteeId;
    this.invitorId = invitorId;

  }
}

File: PersistenceServer/Sep3Blazor-Soap/Sep3Blazor/Model/Invitation.cs
using System;

namespace Sep3Blazor.Model
{
    public class Invitation
    {
        public int id { get; set; }
        public int groupId { get; set; }
        
        public String groupName { get; set; }
        public int inviteeId { get; set; }
        
        public String inviteeName { get; set; }
        public int invitorId { get; set; }
        
        public String invitorName { get; set; }

        public Invitation(int id, int groupId,String groupName, int inviteeId,String inviteeName, int invitorId,String invitorName)
        {
            this.id = id;
            this.groupId = groupId;
            this.groupName = groupName;
            this.inviteeId = inviteeId;
            this.inviteeName = inviteeName;
            this.invitorId = invitorId;
            this.invitorName = invitorName;
        }
    }
}